INSTALL.rst

```diff
@@ -49,12 +56,30 @@ Create a virtual environment and clone the INSPIRE source code using `git`:
 
 .. code-block:: console
 
-    $ mkvirtualenv inspirehep
+    $ mkvirtualenv -p /usr/bin/python2.7 inspirehep
```

Comment:
Ah, right, didn't think about that case... On the other hand in the context of the first installation that shouldn't happen.